modules/EnsEMBL/Web/ZMenu/Transcript/CLS.pm

```diff
@@ -30,6 +30,7 @@ sub content {
   my $self        = shift;
   my $hub         = $self->hub;
   my $object      = $self->object;
+  my @click       = $self->click_location;
```

Comment:
Is this used anywhere in the file?

Reply:
It isn't anymore. Thanks for spotting this one. I'll remove it.